zenodo/modules/records/data/objecttypes.json

```
@@ -712,4 +713,24 @@
     "type": "publication"
   },
   "csl": "article"
+},
+{
+  "$schema": "https://zenodo.org/schemas/records/objecttype-v1.0.0.json",
+  "internal_id": "publication-peerreview",
+  "id": "https://zenodo.org/objecttypes/publication/peerreview#",
+  "title": {
+    "en": "Peer review"
+  },
+  "title_plural": {
+    "en": "Peer reviews"
+  },
+  "schema.org": "https://schema.org/ScholarlyArticle",
+  "datacite": {"general": "Text", "type": "Peer review"},
+  "eurepo": "info:eu-repo/semantics/other",
```

```suggestion
  "eurepo": "info:eu-repo/semantics/review",
```

From https://wiki.surfnet.nl/display/standards/info-eu-repo